cpp/ql/lib/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.0.5
+
```

Comment:
minor: I don't think it looks great to have a blank changelog section. For the codeql-action, we add a section "No user-facing changes." Maybe we should add that here, too.

Reply:
I think that's a good idea, but let's fix that up after we've merged this one back,